CONTRIBUTING.md

```diff
@@ -53,3 +63,13 @@ Sebelum mengerjakan sebuah _issue_, ada hal yang harus diperhatikan:
 1. Apakah ada kontributor lain yang sedang mengerjakannya? Anda dapat mencari apakah ada _pull request_ terkait sebelum memutuskan untuk mulai mengerjakan _issue_ tersebut.
 2. Untuk mulai mengerjakannya, pastikan **_create branch_** dari `main`, lalu segera _commit_ dan _push_ perubahannya **sesegera mungkin** sekecil apapun itu.
 3. Kemudian **buat _pull request_** baru dengan [menjadikannya sebagai **_draft pull request_**](https://github.blog/2019-02-14-introducing-draft-pull-requests/) untuk memberikan sinyal kepada kontributor lainnya bahwa [_pull request_ ini masih sedang dalam pengerjaan](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request). Dengan demikian, orang lain yang ingin mengerjakan _issue_ yang sama akan berpikir dua kali sebelum ikut mengerjakannya.
+
+## Pertanyaan yang sering ditanyakan
+
+### Mengapa kalian menggunakan bahasa Inggris dalam menulis _issue_ dan _pull request_?
+
+Ada beberapa alasan mengapa kita menggunakan bahasa Inggris ketika berkomunikasi di dalam _issue_ dan _pull request_:
+
+1. Secara alamiah, lebih mudah untuk _software engineer_ berkomunikasi dalam bahasa Inggris, karena terminologi-terminologi teknis yang digunakan dalam pemrograman pun berbahasa Inggris. Apabila kita mencoba menerjemahkannya, akan meningkatkan resiko miskomunikasi. Sementara itu, apabila kita tidak menerjemahkan terminologi tersebut, kita harus lebih banyak menggunakan _teks miring_ agar sesuai dengan pedoman-pedoman [PUEBI](https://puebi.js.org/huruf/miring).
+2. Membiasakan para kontributor, yang mayoritas berasal dari Indonesia, untuk berkomunikasi dengan bahasa Inggris. Sangat penting untuk mengasah kemampuan menulis dan membaca bahasa Inggris kita karena sebagian besar dari komunitas _open-source_ di seluruh dunia menggunakan bahasa Inggris sebagai bahasa utama mereka.
+3. Menggunakan bahasa Inggris membuat proyek ini lebih mudah diakui secara global. Apabila kita ingin mendapatkan dukungan dari komunitas global, mereka lebih mudah memahami tujuan kita, sehingga mereka lebih mudah untuk meluangkan waktu dan keahlian mereka untuk membantu membuat proyek ini lebih baik. Contohnya, menyediakan _free credits_ untuk layanan mereka, mengadvokasikan proyek ini kepada pemimpin dunia, atau berkontribusi langsung ke proyek kita.
```

Comment:
Not really sure what "advocating to global leaders" here translates to, or even the context of it (leaders/politicians of the world, or global community leaders?)

Reply:
Both, we're getting more attention in Singapore and I heard they agreed to help us in supplying medical equipments.

And this is another proof that using English attracts attentions from the wider community. 😁

https://twitter.com/zainfathoni/status/1417352680356278277?s=21